docs/sql/decimal.md

```diff
@@ -73,5 +73,9 @@ parsing fails.
     <td><code>LOG10(value)</code></td>
     <td>The logarithm base 10 of value. Produces a runtime error for values less than or equal to zero.</td>
   </tr>
+  <tr>
+    <td><code>SAFE_ADD(left, right)</code></td>
```

Comment:
if you read the documentation for SAFE_ADD from BigQuery you will notice that it actually does not handle short integers. It is a very strange function: https://cloud.google.com/bigquery/docs/reference/standard-sql/mathematical_functions#safe_add. We probably don't want this function in our codebase because I find it very misleading to use, but still, implementing it it is a very good start to learn how this process works.

So my suggestion is to finish this PR making SAFE_ADD work, but we won't do the other functions in the same group (SAFE_SUBTRACT, etc.).